File: php/Product.php
<?php

class Product{
	// atribut private
    private $price;
    private $idProduct;

	// constructor
	public function __construct($price = "", $idProduct = ""){
        $this->price = $price;
        $this->idProduct = $idProduct;;
	}

	// setter/getter
    public function setprice($price){
		$this->price = $price;
	}
    public function getprice(){
		return $this->price;
	}

    public function setidProduct($idProduct){
		$this->idProduct = $idProduct;
	}
    public function getidProduct(){
		return $this->idProduct;
	}
}

?>

File: c++/Product.cpp
#include <iostream>
#include <string>

using namespace std;

class Product{
	private:
	/*atribut private*/
		string price;
		string idProduct;

	public:
		/*constructor*/
		Product(){
			this->price = this->idProduct = "";
		}

		/*getter/setter*/
		void setprice(string price){
			this->price = price;
		}
		string getprice(){
			return this->price;
		}
		void setidProduct(string idProduct){
			this->idProduct = idProduct;
		}
		string getidProduct(){
			return this->idProduct;
		}
		/*destructor*/
		~Product(){}
};